File: wallet.py
class Wallet:
    def __init__(self, balance=0):
        self.balance = balance

    def deposit(self, amount):
        self.balance += amount

    def withdraw(self, amount):
        if amount > self.balance:
            raise ValueError("Insufficient funds")
        self.balance -= amount

    def get_balance(self):
        return self.balance


# Exemple d'utilisation
wallet = Wallet(100)
wallet.deposit(50)
print(wallet.get_balance())  # 150
wallet.withdraw(25)
print(wallet.get_balance())  # 125

File: wallet.java
public class Wallet {
    private int balance;

    public Wallet(int balance) {
        this.balance = balance;
    }

    public void deposit(int amount) {
        this.balance += amount;
    }

    public void withdraw(int amount) {
        if (amount > this.balance) {
            throw new RuntimeException("Insufficient funds");
        }
        this.balance -= amount;
    }

    public int getBalance() {
        return this.balance;
    }
}

// Exemple d'utilisation
Wallet wallet = new Wallet(100);
wallet.deposit(50);
System.out.println(wallet.getBalance()); // 150
wallet.withdraw(25);
System.out.println(wallet.getBalance()); // 125
